web-ui/src/main/resources/catalog/templates/admin/settings/sources.html

```diff
@@ -1,264 +1,275 @@
-<div>
-  <div class="row" data-ng-controller="GnSourcesController">
-    <div class="col-lg-4">
-      <div class="panel panel-default">
-        <div class="panel-heading" data-translate="">manageSources</div>
-        <div class="panel-body">
-          <div class="list-group">
-            <ul class="list-group">
-              <input
-                class="form-control"
-                data-ng-show="sources.length > 1"
-                data-ng-model="sourcesSearch.$"
-                autofocus=""
-                placeholder="{{'filter' | translate}}"
-              />
+<div class="row" data-ng-controller="GnSourcesController" id="gn-sources-container">
+  <div class="col-lg-4">
+    <div class="panel panel-default">
+      <div class="panel-heading" data-translate="">manageSources</div>
+      <div class="panel-body">
+        <div class="list-group">
+          <ul class="list-group">
+            <input
+              class="form-control"
+              data-ng-show="sources.length > 1"
+              data-ng-model="sourcesSearch.$"
+              autofocus=""
+              placeholder="{{'filter' | translate}}"
+            />
 
-              <div class="row">
-                <div class="col-md-3" data-ng-repeat="(k, v) in filter.types">
-                  <label title="{{::('source-' + k) | translate}}">
-                    <input
-                      type="checkbox"
-                      autocomplete="off"
-                      data-ng-model="filter.types[k]"
-                    />
-                    <i
-                      class="fa fa-fw"
-                      title="{{::('source-' + k) | translate}}"
-                      data-ng-class="{'fa-home': k === 'portal',
-                                       'fa-folder': k === 'subportal',
-                                       'fa-cloud': k === 'externalportal',
-                                       'fa-cloud-download': k === 'harvester'}"
-                    ></i>
-                  </label>
-                </div>
+            <div class="row">
```

Comment:
and if you could add 2 class names here: `gn-margin-bottom gn-margin-to`, it would make it look better.